stubs/c-libressl/run.c

```diff
@@ -86,8 +87,8 @@ int main(int argc, char **argv) {
                         break;
                 }
 
-                /* Filter away non-ASCII charaters */
-                if (read_buf[i] < 0x20 || read_buf[i] > 0x7f) {
+                /* Filter away non-printable characters */
+                if (isprint(read_buf[i]) == 0) {
```

Comment:
`isprint` input should be either EOF or representable as an `unsigned char`, otherwise the function's output is undefined. What happens when `read_buf[i]`, which is a `char`, is outside the range 0-127?